### ISSUE TITLE
rvm.io down

### ISSUE THREAD

**Issue by garex** [rvm/rvm#3749]
```
Error 503 Connection refused

Connection refused

Guru Mediation:

Details: cache-bma7025-BMA 1473232071 2452152461

Varnish cache server
```

**Comment by xhoy** [comment]
The fix from #2887 seems to still work.

`curl -sSL https://raw.githubusercontent.com/rvm/rvm/master/binscripts/rvm-installer |bash -s stable`

**Comment by pkuczynski** [comment]
This should work by now...
